src/arkit.cs

```diff
@@ -800,6 +862,12 @@ interface ARSession {
 		[iOS (13,0)]
 		[Export ("updateWithCollaborationData:")]
 		void Update (ARCollaborationData collaborationData);
+
+		[iOS (14, 0)]
+		[Async (ResultTypeName="GeoLocationForPoint")]
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		[Export ("getGeoLocationForPoint:completionHandler:")]
+		void GetGeoLocation (Vector3 position, Action<CLLocationCoordinate2D, double, NSError> completionHandler);
```

Comment:
action with more than 1 (and NSError) arguments should be a custom delegate so the argument names serves as self-documentation